var/spack/repos/builtin/packages/kokkos-legacy/package.py

```diff
@@ -14,6 +14,7 @@ class KokkosLegacy(Package):
     url = "https://github.com/kokkos/kokkos/archive/2.03.00.tar.gz"
     git = "https://github.com/kokkos/kokkos.git"
 
+    version("4.3.00", sha256="53cf30d3b44dade51d48efefdaee7a6cf109a091b702a443a2eda63992e5fe0d")
```

Comment:
This adds a new active kokkos version but all previous versions are deprecated (since legacy). We should not add new versions that's not deprecated. Maybe add a comment at the top saying new versions should go into kokkos itself.

Reply:
Pulled out the new version and added a note telling folks not to make the same mistake I did.